pkg/codegen/internal/test/testdata/output-funcs/getIntegrationRuntimeObjectMetadatum.json

```diff
@@ -134,9 +134,7 @@
         }
       },
       "type": "object",
-      "required": [
-        "type"
-      ]
+      "required": ["type"]
```

Comment:
Is the reformatting in these files a function of your editor?

Reply:
Sorry, yes. It won't happen again. I can undo them if that is helpful.

Reply:
> Sorry, yes. It won't happen again. It will revert in the next commit.

Oh, no worries--I was just wondering if it was intentional. I don't really care about the formatting here as long as we're consistent, and I'm happy to pick whatever approach works most naturally for us when we need to edit there files :)